crates/oxc_linter/src/ast_util.rs

```diff
@@ -170,6 +240,26 @@ impl<'a, 'b> IsConstant<'a, 'b> for Argument<'a> {
     }
 }
 
+impl IsArray for Argument<'_> {
+    fn is_array(&self) -> bool {
+        match self {
+            Argument::ArrayExpression(_) => true,
+            _ => false
+        }
+    }
+}
+
+impl IsNullOrUnderfined for Argument<'_> {
+    fn is_null_or_undefined(&self) -> bool {
```

Comment:
There's already a helper method for this for `Expression`, I think it even has the same name. Can you cast `Argument` to an `Expression`?